src/patternfly/components/Menu/menu.scss

```diff
@@ -246,6 +253,10 @@
     }
   }
   // stylelint-enable
+  .pf-c-divider {
+    --pf-c-select-menu--c-divider--MarginTop: 0;
```

Comment:
Not sure what these do since they're select component styles? Also would you mind leaving a comment to update that var name in a breaking change release?

Reply:
Oh nevermind, I see the example above. I was expecting the fix for https://github.com/patternfly/patternfly/issues/3975 to be in the select component. This fix pairs the menu component with the select component, and as far as I'm aware we haven't decided how we're going to use the menu component with the toggle and paired with other components yet, so it's probably something we should discuss to make sure we're taking the right approach.

Also I forgot to open an issue after we commented out the previous menu scrolling enhancement once we realized that it didn't allow the menu items to scroll while leaving a header or footer static. Here's that issue - https://github.com/patternfly/patternfly/issues/4046